playbooks/templates/common/metadata_common.yaml

```diff
@@ -0,0 +1,10 @@
+metadata          :
+  RPC: {{ maas_rpc | default(true) }}
+  RPC_env_identifier: {{ maas_env_identifier | default('unkown')}}
+  RPC_env_identifier_alt: {{ maas_env_identifier_alt | default('unkown')}}
+  RPC_maas_version: {{  lookup('pipe', 'cd .. && git describe --tags --abbrev=0') }}
+  RPC_deploy_date: {{ ansible_date_time.date }}
```

Comment:
As playbooks are iterative, if we used a more specific timestamp such as epoch, would each template receive a different timestamp or would it be based on the initial execution of the playbook?

Reply:
It will be based on the initial execution of the playbook.